vwifi.c

```diff
@@ -39,6 +44,12 @@ struct owl_ndev_priv_context {
     struct wireless_dev wdev;
 };
 
+static char *ssid_list = "";
+module_param(ssid_list, charp, 0644);
+MODULE_PARM_DESC(ssid_list, "Self-defined SSIDs.");
+static const char sdssid_delim[] = "]";
+static const char default_ssid_list[] = DEFAULT_SSID_LIST;
```

Comment:
Why did you specify another variable `default_ssid_list`? Can you eliminate it?

Reply:
fixed